### DISCUSSION TITLE
keep a pane, but swap another one?

### DISCUSSION THREAD

**Discussion by dmd** [zellij-org/zellij#3140] [General]
Suppose I want to split my screen into a "left" and "right" pane. I want the one on the right to stay the same (maybe I'm tailing a logfile there), but I want to switch the one on the left between various different servers.

Is there a way to do this **without** nesting zellij sessions?

**Comment by imsnif** [comment]
Have you considered using the "stacked" layout? You can get to it by opening 3 or more panes and then switching to it with `Alt` + `[` until you get there.

**Comment by dmd** [comment]
Oh that is nice! That isn't *precisely* what i wanted but it's certainly close enough.
